moninag/static/src/ts/app/checks/check-add.component.ts

```diff
@@ -0,0 +1,82 @@
+import { Component } from '@angular/core';
+import { Location } from '@angular/common';
+
+import { Check } from './check';
+import { ChecksService } from './checks.service';
+import { Service } from '../services/services';
+
+
+@Component({
+    selector: 'checkadd-app',
+    template: `
+    <div class="container">
```

Comment:
move template section into the separate file `check-add.component.html` and require it like this: 
`template: require('./check-add.component.html')`